tools/db_bench_tool.cc

```diff
@@ -932,6 +934,68 @@ DEFINE_uint64(
     "If non-zero, db_bench will rate-limit the writes going into RocksDB. This "
     "is the global rate in bytes/second.");
 
+// the parameters of mix_graph
+DEFINE_double(key_dist_a, 0.0,
+              "The parameter 'a' of key access distribution model "
+              "f(x)=a*x^b");
+DEFINE_double(key_dist_b, 0.0,
+              "The parameter 'b' of key access distribution model "
+              "f(x)=a*x^b");
+DEFINE_double(value_theta, 0.0,
```

Comment:
Wondering if value_theta and iter_theta can be combined into a single paramater?

Reply:
The value distribution and the iterator length distribution fit to the same General Pareto distribution model. But the parameters are different. So we cannot combine them.